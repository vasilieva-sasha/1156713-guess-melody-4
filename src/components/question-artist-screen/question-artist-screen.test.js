import React from "react";
import renderer from "react-test-renderer";
import QuestionArtistScreen from "./question-artist-screen";

const question = {
  type: `artist`,
  song: {
    artist: `Jim Beam`,
    src: `https://upload.wikimedia.org/wikipedia/commons/4/4e/BWV_543-fugue.ogg`,
  },
  answers: [{
    picture: `https://api.adorable.io/avatars/128/0.6174712697143883`,
    artist: `John Snow`,
  }, {
    picture: `https://api.adorable.io/avatars/128/0.6174712697143883`,
    artist: `John Snow`,
  }, {
    picture: `https://api.adorable.io/avatars/128/0.6174712697143883`,
    artist: `John Snow`,
  }],
};

describe(`QuestionArtistScreenComponent`, () => {
  it(`Render QuestionArtistScreen`, () => {
    const tree = renderer
      .create(
          <QuestionArtistScreen onAnswer={() => {}} question={question} />
      )
      .toJSON;

    expect(tree).toMatchSnapshot();
  });
});
