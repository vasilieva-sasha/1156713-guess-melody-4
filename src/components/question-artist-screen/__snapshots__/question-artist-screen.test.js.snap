// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionArtistScreenComponent Render QuestionArtistScreen 1`] = `[Function]`;
